### ISSUE TITLE
v1.1.0 error on Mac (x64) "The application can't be opened."

### ISSUE THREAD

**Issue by brazzam** [CDeenen/MaterialServer#10]
I'm getting a "can't be opened" message when trying to run the server application on my Mac when opening from anywhere, including the Applications folder.

<img width="285" alt="Screenshot 2023-05-30 at 3 16 40 PM" src="https://github.com/CDeenen/MaterialServer/assets/114369198/ef66750e-e1d1-4795-a19e-38c3b7c77691">

_**System:** 2019 Mac Pro | 3.2 GHz 16-Core Intel Xeon W | macOS 13.4 (Ventura)_

**Comment by CDeenen** [comment]
Let me start out by saying that I am not at all familiar with osx, I'm basing what I'm saying on what I've heard others say.
Any osx related issues seem to be solved by [building the app yourself from the source](https://github.com/CDeenen/MaterialServer/wiki/Running-and-Building-From-the-Source).
However, looking at this error: are you sure you downloaded the right osx package? For an Inten cpu you'd need to use the macos-x64 package.

**Comment by brazzam** [comment]
Yes, I downloaded the x64 version.  I've uninstalled, redownloaded, and reinstalled the app about 3 times now to the same result.

I will try to learn how to build the app myself, but I am not very versed at command line stuff, so I will want to learn more before fooling around in there too much.  In the meantime, I'll just have to get by without I suppose.

**Comment by gludington** [comment]
The Electron Framework is not present in the materialserver-macos-x64.zip application archive, where it is in the M1 version.  x64 MaterialServer cannot start then unless the user happens to have the Electron Framework installed elsewhere on their path.  I can confirm when I npm run package locally on my Intel OSX machine, the framework is included in the resulting build output, so I cannot say why that is not the case for the one on the release page.

**Comment by QcGreywolf** [comment]
Any solution to this that doesn't include installing untold GB of software to compile the source code? It would be my absolute last resort. I run foundry on a music production machine and would prefer to have a already compiled package so I don't have to clutter up my machine with the mountain of files that come with installing 3 different development environments.

**Comment by CDeenen** [comment]
I've been working on a replacement for Material Server, called Material Companion. I've heard that it has less issues for osx users. I'm currently in the last stages of testing before I release a version that's compatible with Material Deck. Material Keys compatibility will come a bit later.
